.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,26 +13,26 @@ about: Create a report to help us improve
 
 ---
 
-## :writing_hand: Describe the bug
```

Comment:
Do we have to remove the emojis? ☹️

Reply:
Well, main thing with this alignment was to have similar/same things as open source template. And reason is partially in future if open source template changes it will be easier to align our template.

I agree emojis give certain flavor to files, but I am not sure if we want to differentiate from other teams in that regard

Reply:
I returned emojis, but PR template will stay clear 😂

Reply:
I would even suggest to Majcen that we add the emojis to the template TBH, it's nice to have a bit of flavor 😄